### ISSUE TITLE
Unknown clippy lint: clippy::unwrap_used

### ISSUE THREAD

**Issue by c410-f3r** [rust-lang/rust-clippy#5615]
Same thing for `expect_used`. The following:

```rust
#![forbid(clippy::unwrap_used)]

fn main() {
    let a = Some(1);
    let _ = a.unwrap();
}
```

Throws:

```
warning: unknown clippy lint: clippy::unwrap_used
 --> src/main.rs:1:11
  |
1 | #![forbid(clippy::unwrap_used)]
  |           ^^^^^^^^^^^^^^^^^^^
  |
  = note: `#[warn(clippy::unknown_clippy_lints)]` on by default
  = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#unknown_clippy_lints
```

Happened when using Rustup's Clippy (`clippy 0.0.212 (a74d186 2020-05-14)`) and also happens at the playground (https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=b80d5e2a624402628c7eaf0295e99b25). Not sure if this behavior is intentional

**Comment by c410-f3r** [comment]
Ah NVM, this is on master only

**Comment by flip1995** [comment]
should be in todays (2020-05-19) nightly
